server/auth/github/client.go

```diff
@@ -173,6 +177,29 @@ func (c *Client) IsMember(organization, token string) (bool, error) {
 	return true, nil
 }
 
+// IsTeamMember returns true if the given user is a member of the team.
+func (c *Client) IsTeamMember(teamId, name string, token string) (bool, error) {
```

Comment:
what if we added:
```
type User struct {
	Name        string `json:"name"`
	GitHubToken string `json:"-"`
    GitHubLogin string `json:"github_login"`
}
```

or some such thing.

I think the current flow of getting the user in `authenticate` and then having all github authorizers reference values from the empire user makes sense.